moosefs_tricorder/common.py

```diff
@@ -68,12 +70,12 @@ def load_chunkserver_metrics(moosefs_master: str) -> dict:
     return data
 
 
-def load_master_metrics(moosefs_master) -> dict:
+def load_master_metrics(moosefs_master: str, moosefs_master_port: int) -> dict:
```

Comment:
Set `moosefs_master_port` to default to 9420 for consistency with `load_chunkserver_metrics()`